src/content/platform-integration/web/wasm.md

```diff
@@ -85,7 +63,7 @@ At the bottom of the output, you should find experimental Wasm options like:
 
 ```console
 Experimental options
-    --wasm                       Compile to WebAssembly rather than JavaScript.
+    --wasm                       Compile to WebAssembly (with fallback to JavaScript).
                                  See https://flutter.dev/wasm for more information.
```

Comment:
This link is linking to the same page that we're on... Can we delete?

Reply:
This is SHOWING what the output of the CLI is, so I think it's fine.